system_metrics_collector/src/system_metrics_collector/proc_pid_cpu_data.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <sstream>
+#include <string>
+
+#include "proc_pid_cpu_data.hpp"
+
+
+namespace system_metrics_collector
+{
+
+/*static*/ constexpr const uint64_t ProcPidCpuData::kEmptyData;
```

Comment:
Is the comment left here for a reason?

Reply:
It's to indicate that this is the definition of a static member variable. It seems to be the convention in this project.

Reply:
I'm hoping we adopt it in other projects as well. This came out of an in person review with @thomas-moulard and prefer this to understand context.